File: Java/Car.java
package Java;

//Clase para definir los tipos de vehiculos Uber que existen
public class Car {
    Integer id;
    String license;
    Account driver;
    Integer passengers;

    public Car(String license, Account driver){ //Parametros declarados de manera local
        this.license = license;
        this.driver = driver;

    }
    
    void printDataCar(){
        System.out.println("License: " + license + "Driver: " + driver.name + " Document: " + driver.document);
    }
}

File: JS/Car.js
//Clase para definir los tipos de vehiculos Uber que existen
class Car {
    constructor(license, driver) {
        this.id;
        this.license = license;
        this.driver = driver;
        this.passengers;
    }
    printDataCar() {
        console.log(this.driver);
        console.log(this.driver.name);
        console.log(this.driver.document);

    }
}

